src/SecurityUtilitiesPackageReference/SecurityUtilitiesApiUtilizationExample/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Security.Utilities;
+
+namespace SecurityUtilitiesApiUtilizationExample
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int iterations = 10;
+
+            var masker = new IdentifiableScan(WellKnownRegexPatterns.HighConfidenceMicrosoftSecurityModels,
+                                              generateCorrelatingIds: false);
+
+            foreach (var pattern in WellKnownRegexPatterns.HighConfidenceSecurityModels)
+            {
+                var identifiable = pattern as IIdentifiableKey;
+                if (identifiable == null) { continue; }
+
+                foreach (ulong seed in identifiable.ChecksumSeeds)
+                {
+                    for (int i = 0; i < iterations; i++)
+                    {
+                        foreach (string signature in identifiable.Signatures!)
+                        {
+                            string key = IdentifiableSecrets.GenerateStandardBase64Key(seed,
+                                                                                     identifiable.KeyLength,
+                                                                                     signature);
+
+                            int found = masker.DetectSecrets(key).Count();
```

Comment:
>int found = masker.DetectSecrets(key).Count() [](http://example.com/codeflow?start=28&length=45)

you don't have any validation of function in your test.

neither do you return or check a success code. Maybe a non-zero result is set for an unhandled exception in your script? Will go look.

My specific suggestion is that you verify we actually find secrets here, a non-zero count is probably sufficient.

Please don't block on this note, let's get this in and you can refine.

Reply:
@michaelcfanning I specifically don't want to perform any functional test here. Shouldn't the objective here be limited to testing whether the nuget loads and operates on relevant frameworks and platforms?